modules/installation-initializing-manual.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+[id="installation-initializing-manual-{context}"]
+= Manually creating the installation configuration file
+
+For some User Provisioned Infrastructure (UPI) installations of {product-title}
+you must manually generate your installation configuration file.
+
+.Prerequisites
+
+* Obtain the {product-title} installation program and the access token for your
+cluster.
+
+.Procedure
+
+. Create an installation directory to store your required installation assets
+in:
++
+----
+$ mkdir <installation_directory>
+----
+
+. Customize the following `install-config.yaml` file template and save the
+it in the `<installation_directory`.
++
+[NOTE]
+====
+You must name this configuration file `install-config.yaml`.
+====
+
+. Back up the `install-config.yaml` file so that you can use it to install
+multiple clusters.
++
+[IMPORTANT]
+====
+The `install-config.yaml` file is deleted during the next step of the
```

Comment:
nit: `deleted` -> `consumed`
installer just consumed it and stores it in internal state. and then removes that file from disk.
running `create install-config` will return the install-config to disk.

Reply:
"Consumed" is one of the terms that our style guide suggests avoiding if other terms are clear enough, but it does seem appropriate here. Thanks!

@abhinavdahiya, do you know at what point you can no longer run `create install-config` to regenerate the install-config, and is there any reason why you wouldn't want to do it?